Mux-Stats-Google-IMA.podspec

```diff
@@ -25,11 +25,11 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '10.0'
 
 
-  s.ios.dependency 'Mux-Stats-AVPlayer', '~> 2.0'
+  s.ios.dependency 'Mux-Stats-AVPlayer', '~> 2.10'
```

Comment:
We haven't released this yet but we need to update the example project in 2.10 avPlayer to point to this pod version as well so I think it makes more sense to release this one first

Reply:
🤔  Will this SDK build if we specify a dependency that doesn't yet exist?